grunt/tasks/release.js

```diff
@@ -134,8 +107,10 @@ function msg() {
   var steps = [
     'Still todo:',
     '* put files on CDN',
+    '* add starter pack (git add -f docs/downloads/react-version.zip)',
```

Comment:
Yea but I figured the people doing this (me and you) are smart enough to know that. This isn't meant to be a complete step by step but a gentle reminder